client/src/components/Tip/tests/Tip-story.js

```diff
@@ -44,8 +45,9 @@ storiesOf('Admin/Tip', module)
               fieldTitle={'Tip Field'}
               id={'tip-field'}
               content={text('Content', 'Example tip contents')}
-              icon={selectV2('Icon (examples)', ['lamp', 'attention', 'flag'], 'lamp')}
+              icon={selectV2('Icon (examples)', ['lamp', 'menu-help', 'attention', 'flag'], 'lamp')}
               importance={selectV2('Importance', importanceLevels, 'normal')}
+              type={selectV2('Type', TIP_TYPES, TIP_TYPES.TEXT_FIELD)}
```

Comment:
Given how this story is structured, having this prop here doesn't provide any value. 

We should have a tip on the label and one on the field. Maybe we need a different story for each use case. Or maybe this should be on a story specific to form fields.

Reply:
I've split the two different types to be different things in the pattern-lib